index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <title>Website Design Project</title>
+        <link rel="stylesheet" href="style.css" type="text/css">
+    </head>
+    <body>
+        <header class="imagecontainer">
+            <section class="img defineimage"></section>
```

Comment:
```suggestion
            <div class="defineimage"></div>
```
A section tag is a nice HTML semantic tag. Thank you for attempting to use HTML 5 semantic tags.
However, a section should not be used for such a small area of code for example just storing an image.
For images, you can either use a simple div for a background image or just an img tag to bring the image here directly through its src attribute.

Also, please you do not need to add 
```
class = "img"
```
Why? You are not styling img in your CSS.

Reply:
I have replaced the section with div and have removed the class tag inside it. Thank you very much for pointing it out.